common/libuavcan.cmake

```diff
@@ -0,0 +1,26 @@
+set(LIBUAVCAN ${COMMON}/uavcan_dsdl/libuavcan)
+set(UDEFS ${UDEFS}
+  -DUAVCAN_STM32_CHIBIOS=1
+  -DUAVCAN_STM32_NUM_IFACES=1
+  -DUAVCAN_STM32_TIMER_NUMBER=6
+)
```

Comment:
I think this is very good already. I would suggest of using `target_compile_definitions(${EXECUTABLE} ...)` for setting up the definitions for UAVCAN instead of using `UDEFS`

Reply:
Done